README.md

```diff
@@ -507,7 +507,7 @@ $logomark_svg_markup = am_get_symbol(
 
 ## Requirements
 
-* WordPress: 5.2.0+
+* WordPress: 6.3.0+
```

Comment:
Updating the required version from 5.2 to 6.3 is significant. Therefore I think the version change of the plugin itself should reflect this as it will be a "breaking" change for sites that may be on 6.2 or below.

So the plugin version could be bumped higher, a major version change, instead of a minor version change OR you could refactor to provide backwards compatibility for versions prior to WP 6.3.

Reply:
I agree, that is a big leap from 5.2 to 6.3. After a few others chimed in on Slack about it, backwards compatibility is the way!